src/view/shell/NavSignupCard.tsx

```diff
@@ -44,12 +44,12 @@ let NavSignupCard = ({}: {}): React.ReactNode => {
       <View style={[a.flex_row, a.flex_wrap, a.gap_sm, a.pt_md]}>
         <Button
           onPress={showCreateAccount}
-          label={_(msg`Sign up`)}
+          label={_(msg`Create account`)}
           size="small"
           variant="solid"
           color="primary">
           <ButtonText>
-            <Trans>Sign up</Trans>
+            <Trans>Create account</Trans>
```

Comment:
Can we specify an `id` for Lingui so that this string doesn't change in other langs?